### ISSUE TITLE
Timeout issue

### ISSUE THREAD

**Issue by Spanners94** [risinek/esp32-wifi-penetration-tool#72]
I know its normal for it to time out but when it does and i reconnect to the management AP it still says timeout without any infomation as if to say it didnt do anything. I do it on my home network and theres nothing in place to stop attacks

**Comment by risinek** [comment]
Have you tried refreshing the page after you reconnect?

**Comment by Spanners94** [comment]
Yeah I have. I looked at the serial monitor and it seems to be not sending any packets

**Comment by Spanners94** [comment]
![PXL_20231105_093254538.MP.jpg](https://github.com/risinek/esp32-wifi-penetration-tool/assets/95079147/9b3b65dc-56a8-4c53-b4be-2d30e1716e98)

**Comment by risinek** [comment]
those logs are not from this project. or you mean you were using another esp32 to monitor network to see deauth packets from this project?
